src/background/intial-assessment-store-data-generator.ts

```diff
@@ -45,8 +46,10 @@ export class InitialAssessmentStoreDataGenerator {
         return state as IAssessmentStoreData;
     }
 
-    private constructInitialDataForAssessment(persistedTests: { [key: string]: IAssessmentData } = null) {
-        const assessmentData: { [key: string]: IAssessmentData } = {};
+    private constructInitialDataForAssessment(
+        persistedTests: { [key: string]: IAssessmentData } = null,
```

Comment:
>{ [key: string]: IAssessmentData } [](start = 24, length = 34)

(per offline discussion) Consider also using `DictionaryStringTo` here

Reply:
Actually, I think following the example of `constructGeneratedAssessmentInstancesMap` and giving a better name to this type to indicate what the key is supposed to be exactly would be even better

---
In reply to: [266618051](https://github.com/Microsoft/accessibility-insights-web/pull/375#discussion_r266618051) [](ancestors = 266618051)

Reply:
changed both instances to `DictionaryStringTo<IAssessmentData>`